pkg/agent/agent.go

```diff
@@ -99,6 +99,21 @@ func (i *Initializer) setupOVSBridge() error {
 	}
 	i.nodeConfig.BridgeName = i.ovsBridgeClient.GetBridgeName()
```

Comment:
Move line 100 to initNodeLocalConfig() - it should be there!

Reply:
Done